### PR TITLE
WS-113: Added member details population

### PR DESCRIPTION
**Pull request by Eyrim**

### CODE REVIEW

File: src/main/java/com/whalespottingjava/controllers/MembersController.java [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
+        Authentication authentication = securityContext.getAuthentication();
+        MemberDetails memberDetails = (MemberDetails) authentication.getPrincipal();
+        Long memberId = memberDetails.getMember().getId();
+        Member member = this.memberDetailsService.loadMemberById(memberId);
+
+        model.addAttribute("member", member);
+
         return "my_account";
     }
 }
```
Comment (CadfanWilliams) [inline src/main/java/com/whalespottingjava/controllers/MembersController.java:36]: Need new line at end of line

File: src/main/java/com/whalespottingjava/services/MemberDetailsService.java [hunk 2]
```diff
@@ -28,4 +29,12 @@ public UserDetails loadUserByUsername(String username) {
...
     }
+
+    public Member loadMemberById(long id) {
+        return memberRepository
+                .findById(id)
+                .orElseThrow(
+                        EntityNotFoundException::new
+                );
+    }
 }
```
Comment (CadfanWilliams) [inline src/main/java/com/whalespottingjava/services/MemberDetailsService.java:40]: New Line at end of file

**Review by CadfanWilliams — CHANGES_REQUESTED** [review]
VERY VERY VERY miniscule changes but other than that looks good.

File: src/main/java/com/whalespottingjava/services/MemberDetailsService.java (hunk 2 above)
Comment (Eyrim) [reply src/main/java/com/whalespottingjava/services/MemberDetailsService.java:40]: There is one at the end of the file, git just doesn't show it. I believe the sign is the no-entry type symbol at the end of the file on the diff

File: src/main/java/com/whalespottingjava/controllers/MembersController.java (hunk 1 above)
Comment (Eyrim) [reply src/main/java/com/whalespottingjava/controllers/MembersController.java:36]: There is one at the end of the file, git just doesn't show it. I believe the sign is the no-entry type symbol at the end of the file on the diff

**Review by CadfanWilliams — APPROVED** [review]

**Pull request merged**